src/manifest.cpp

```diff
@@ -606,8 +606,8 @@ manifest_dump(const std::string& path, FILE* stream)
     char name[DIGEST_STRING_BUFFER_SIZE];
     fmt::print(stream, "  {}:\n", i);
     fmt::print(stream, "    File info indexes:");
-    for (unsigned j = 0; j < mf->results[i].file_info_indexes.size(); ++j) {
-      fmt::print(stream, " {}", mf->results[i].file_info_indexes[j]);
+    for (unsigned int& file_info_indexe : mf->results[i].file_info_indexes) {
```

Comment:
`indexe` -> `index`
`unsigned int&` -> `uint32_t`

Also, I think that we should avoid shadowing the outer loop variable.

Reply:
Not sure about the shadowing comment? Outer loop variable is `i`.

Reply:
> Not sure about the shadowing comment? Outer loop variable is `i`.

Right, strike that – I misinterpreted the narrow diff view.